app/workers/facilities/pssg_download.rb

```diff
@@ -31,28 +31,39 @@ def create_and_save_drive_time_data(drive_time_data)
       name = attributes&.dig('Name')
       drive_time_band = facility.drivetime_bands.find_or_initialize_by(vha_facility_id: id, name: name)
       drive_time_band.unit = 'minutes'
-      drive_time_band.min = attributes&.dig('FromBreak')
-      drive_time_band.max = attributes&.dig('ToBreak')
+      drive_time_band.min = round_band(attributes&.dig('FromBreak'))
+      drive_time_band.max = round_band(attributes&.dig('ToBreak'))
       drive_time_band.name = name
       drive_time_band.polygon = extract_polygon(drive_time_data)
       drive_time_band.save
       facility.save
     end
 
+    def round_band(band)
+      if (band % 10).zero?
+        band
+      else
+        (band.to_i / 10) * 10
+      end
+    end
+
     def extract_polygon(drive_time_data)
       rings = drive_time_data&.dig('geometry', 'rings')
+      return if rings.blank?
```

Comment:
don't you want to check if this is blank in the `create_and_save_drive_time_data` so you can skip the whole building up of `drive_time_band` and even the `facility.save` and `drive_time_band.save`?